en/admin/admin_utils.rst

```diff
@@ -790,9 +790,9 @@ Reference to the object deleted during compacting is displayed as **NULL**, whic
 
 *   *database_name*: The name of the database whose space is to be compacted. The path name to the directory where the database is to be created must not be included.
 
-*   *class_name_list*: You can specify the list of tables names that you want to compact space after a database name; the **-i** option cannot be used together. It is used in client/server mode only.
+*   *class_name_list*: You can specify the list of tables names that you want to compact space after a database name; the **-i** option cannot be used together. If you use the lists on client/server mode, It is skipped freeing objects such as catalog, delete files and tracker etc. 
```

Comment:
It is skipped freeing objects such as catalog, delete files and tracker etc.
-->
freeing (cleanup of?) objects such as catalog, delete files and tracker, etc. is not performed.
or 
it skips freeing (cleanup of?) objects such as catalog, delete files and tracker, etc.

Reply:
If you don't sure about "freeing",  how about that i change from "freeing objects" to "securing space taken objects".
and I would be better no 2 than no 1.

Reply:
"securing space taken objects" might be more precise :), and it would be better adding 'by' before objects. 
 
if no.2 is taken into consideration, the sentence would change to: 
it skips securing space taken by objects such as catalog, delete files and tracker, etc.

Reply:
OK, i will modify as your opinion.